src/sonic-host-services/scripts/process-reboot-cause

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
```

Comment:
Please change shebang to `#!/usr/bin/env python3` and ensure script is Python 3-compliant.